website/pages/use-cases/identity-based-access/index.jsx

```diff
@@ -41,149 +49,12 @@ export default function DataEncryptionUseCase() {
         </div>
       </section>
 
-      {/* Features / Text and content */}
-      <section className="g-container">
-        <SectionHeader headline="Identity-based Access Features" />
-
-        <div className="g-text-and-content">
-          <div className="text">
-            <div>
-              <h3 id="secure-plugins">Identity Plugins</h3>
-              <p>
-                Improve the extensibility of Vault with pluggable identity
-                backends
-              </p>
-            </div>
-          </div>
-          <div className="content logo-grid">
-            <ul className="g-logo-grid large">
-              {[
-                'aws',
-                'azure',
-                'gcp',
-                'kubernetes',
-                'nomad',
-                'okta',
-                'pivotalcf',
-                'ssh',
-              ].map((logo) => (
-                <li key={logo}>
-                  <img
-                    src={require(`./img/logos/${logo}.png`)}
```

Comment:
I've moved some previously project-local images that were being `require`'d to Dato. In some cases this does result in a URL for the `<img />` `src` where previously our `next-optimized-images` was using a `base64` data `src`. This felt like an acceptable tradeoff to make content & markup a bit more separate and clear.